e2e-tests/tests/major-upgrade/11-read-from-primary.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+timeout: 30
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+      data=$(run_psql_local '\c myapp \\\ SELECT * from myApp;' "postgres:$(get_psql_user_pass major-upgrade-pguser-postgres)@$(get_psql_user_host major-upgrade-pguser-postgres)")
+
+      kubectl create configmap -n "${NAMESPACE}" 11-read-from-primary --from-literal=data="${data}"
```

Comment:
@tplavcic @egegunes maybe we need to add backup and restore stops to be sure that this basic functionality works after the major update.